src/components/Search/SearchPresentation.js

```diff
@@ -60,7 +59,11 @@ function SearchPresentation({
 
   return (
     <>
-      <section className="bg-gradient-to-b from-word to-word-dark p-5">
+      <section
+        className={`${
+          wholeDomain ? 'bg-bgRed' : 'bg-gradient-to-b from-word to-word-dark'
```

Comment:
Please let me know if another colour should be used for the red search background, this was just my inference